src/components/Textfield.tsx

```diff
@@ -1,18 +1,16 @@
-import React, { EventHandler, FormEvent, useState } from "react";
-import UiTextfield from "@ied/ui-textfield";
+import React, { useState } from 'react'
+import UiTextfield from '@ied/ui-textfield'
 
-const Textfield = () => {
-  const [value, setInputValue] = useState<string>();
+const Textfield: React.FC = () => {
+  const [value, setInputValue] = useState<string>()
```

Comment:
Convention breake:

```suggestion
  const [value, setValue] = useState<string>()
```

or

```ts
 const [inputValue, setInputValue] = useState<string>()
```